src/css/index.js

```diff
@@ -12,12 +12,31 @@ const exp = {
     TEXT_GREY: 'rgba(0,0,0,.87)',
     TEXT_DARK: '#000000',
     WHITE: '#FFFFFF',
+    INVISIBLE: 'rgba(0, 0, 0, 0)'
   },
   sizes: {
     ICON: '24px',
     'fixed-table-header-height': '28px',
     'fixed-table-col-width': '80px',
   },
+  typography: {
+    size: {
+      xxSmall: '10px',
+      xSmall: '12px',
+      small: '14px',
+      medium: '16px',
+      large: '20px',
+      xLarge: '32px',
+    }
```

Comment:
Probably more correct to use "rem" but font sizes can be tricky so I've kept them at px for now.  Until they make 4k phones, we'll be fine.